### PR TITLE
lightning: update lightning example binary version and download link

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Might cause conflicts after applied to another branch
  - The TiDB download page link is different between v3.0 and v5.0

### CODE REVIEW

**Comment by glorv** [comment]
@qiancai @kennytm @3pointer PTAL

File: get-started-with-tidb-lightning.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，
 
 ### 第 1 步：部署 TiDB 集群
 
-在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群（版本要求 2.0.9 以上），本教程使用 TiDB 4.0.3 版本。部署方法可参考 [TiDB 部署方式](https://docs.pingcap.com/zh/tidb/v3.0/overview#部署方式)。
+在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群（版本要求 2.0.9 以上），本教程使用 TiDB v5.0.0 版本。部署方法可参考 [使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
```
Comment (qiancai) [inline get-started-with-tidb-lightning.md:44]: 因为此文档是针对 v5.0 的， 所以这里的“（版本要求 2.0.9 以上）”可以删除。

```suggestion
在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群，本教程使用 TiDB v5.0.0 版本为例。部署方法可参考 [使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
```

Comment (glorv) [reply get-started-with-tidb-lightning.md]: 那 master 文档需要保留么？🤔

Comment (qiancai) [reply get-started-with-tidb-lightning.md]: master 代表正在开发的分支，就是 stable 以上的分支，所以也不需要保留。更多关于删除历史版本信息描述的讨论，可以参考 https://github.com/pingcap/docs/issues/4549 哈。

File: get-started-with-tidb-lightning.md [hunk 2]
```diff
@@ -41,13 +41,13 @@ TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，
 
 ### 第 1 步：部署 TiDB 集群
 
-在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群（版本要求 2.0.9 以上），本教程使用 TiDB 4.0.3 版本。部署方法可参考 [TiDB 部署方式](https://docs.pingcap.com/zh/tidb/v3.0/overview#部署方式)。
+在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.0.0 版本为例，具体部署方法可参考 [使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
 
 ### 第 2 步：下载 TiDB Lightning 安装包
 
-通过以下链接获取 TiDB Lightning 安装包（选择与 TiDB 集群相同的版本）：
+通过以下链接获取 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群， 建议选择最新稳定版本）：
```
Comment (qiancai) [inline get-started-with-tidb-lightning.md:48]:
```suggestion
通过以下链接获取 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）：
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ aliases: ['/docs-cn/dev/tidb-lightning/deploy-tidb-lightning/','/docs-cn/dev/ref
 
 #### 第 2 步：下载 TiDB Lightning 安装包
 
-在[工具下载](/download-ecosystem-tools.md#tidb-lightning)页面下载 TiDB Lightning 安装包（需选择与 TiDB 集群相同的版本）。
+在[工具下载](/download-ecosystem-tools.md#tidb-lightning)页面下载 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群， 建议选择最新稳定版本）。
```
Comment (qiancai) [inline tidb-lightning/deploy-tidb-lightning.md:86]:
```suggestion
在[工具下载](/download-ecosystem-tools.md#tidb-lightning)页面下载 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）。
```

File: get-started-with-tidb-lightning.md [hunk 4]
```diff
@@ -41,13 +41,13 @@ TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，
 
 ### 第 1 步：部署 TiDB 集群
 
-在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群（版本要求 2.0.9 以上），本教程使用 TiDB 4.0.3 版本。部署方法可参考 [TiDB 部署方式](https://docs.pingcap.com/zh/tidb/v3.0/overview#部署方式)。
+在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.0.0 版本为例，具体部署方法可参考 [使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
```
Comment (qiancai) [inline get-started-with-tidb-lightning.md:44]:
```suggestion
在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.0.0 版本为例，具体部署方法可参考[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
```

File: get-started-with-tidb-lightning.md [hunk 5]
```diff
@@ -41,13 +41,13 @@ TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，
...
-在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群（版本要求 2.0.9 以上），本教程使用 TiDB 4.0.3 版本。部署方法可参考 [TiDB 部署方式](https://docs.pingcap.com/zh/tidb/v3.0/overview#部署方式)。
+在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.0.0 版本为例，具体部署方法可参考[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
 
 ### 第 2 步：下载 TiDB Lightning 安装包
 
-通过以下链接获取 TiDB Lightning 安装包（选择与 TiDB 集群相同的版本）：
+通过以下链接获取 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）：
 
-- **v4.0.3**: [tidb-toolkit-v4.0.3-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v4.0.3-linux-amd64.tar.gz)
+- **v5.0.0-nightly**: [tidb-toolkit-v5.0.0-nightly-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v5.0.0-nightly-linux-amd64.tar.gz)
```
Comment (TomShawn) [inline get-started-with-tidb-lightning.md:50]: @glorv can the nightly version be used for production?

Comment (glorv) [reply get-started-with-tidb-lightning.md:50]: There is no `5.0.0` version available now. So maybe we need to merge this PR after `5.0.0-ga` version is released?

Comment (TomShawn) [reply get-started-with-tidb-lightning.md:50]: OK

Comment (yikeke) [reply get-started-with-tidb-lightning.md:50]:
```suggestion
- **v5.0.0**: [tidb-toolkit-v5.0.0-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v5.0.0-linux-amd64.tar.gz)
```
my suggestion after 5.0 GA ⬆️

**Review by yikeke — APPROVED** [review]

**Pull request merged**